src/ducks/classifications.js

```diff
@@ -186,7 +186,9 @@ const submitClassification = () => {
       }
       //Log
       console.log('Submit classification: Success');
-      Split.classificationCreated(classification);
+      try {  //Fix: IE11 doesn't know what to do with Split.classificationCreated()
+        Split.classificationCreated(classification);
```

Comment:
If merged, I'm not sure this should be a permanent fix. It's important we have this `.classificationCreated` function to track classifications through seven-ten. I have a suspicion there could be a JS function somewhere in `classificationCreated()` that isn't supported in IE? However, I feel like this should appear somewhere in the log.